### DISCUSSION TITLE
NEWS downloader can't create epubs after last update (errors)

### DISCUSSION THREAD

**Discussion by pyromaker** [koreader/koreader#8939] [General]
Hello, after updating koreader to the latest version it can't create anymore epubs when syncing news.
I am using only 1 feed in the settings. 

After downloading the articles when it's time to convert the articles to epub,  from 50 articles the "successfully" converted EPUBS are 7-10. After the "converting to EPUB 1/50 window"when it's done I manually press Continue in every error message of EPUB "Error building EPUB... (header of the article)"

My settings feed are

Download full article true
Include Images false
Enable filter true
Filter element div.release-main-section (this was by default I think it's something new in the last update?)
Volumize feed false .

Is this something like a bug? Thanks

**Comment by Frenzie** [comment]
I somewhat overlooked it for the release but it's probably the same thing as the discussion is in https://github.com/koreader/koreader/issues/8867

**Reply by pyromaker** [reply]
Thanks a lot, hope will be fixed. Love your program
